Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -15,7 +15,8 @@ msgstr ""
 
 #. translators: Subtitle to be displayed below the application name in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_subtitle"
-msgid "Simple notes, todos and memos"
+msgid "Simple notes, todos and memos
+"
```

Comment:
I think this glitch is a known issue in our release toolkit, at least I've seen it happen a couple of times in the past (and to my knowledge, so far, RMs who encountered it fixed it manually, we definitively need to investigate and solve it at its source in the toolkit)

This is the reason why your `.pot` is invalid (considered a syntax error) and was not imported in GlotPress

```suggestion
msgid "Simple notes, todos and memos"
```